content/guides/pedestal-with-component.adoc

```diff
@@ -1,16 +1,419 @@
+----
+We'll implement a single route, _/greet_, using Pedestal's Tabular
+Routing syntax.
+
+Now that we've got our Pedestal Component and routes, we can wire
+them up in a Component system map.
+
+Create a `system.clj` file. This file will contain our system map and
+system constructor.
+
+[source, clojure]
+.src/system.clj
+
+----
+include::component/src/system.clj[tags=ns]
+----
+<1> Require `com.stuartsierra.component`. It will be used to  to
+create the Component system map.
+<2> Require `reloaded.repl` for it's system management functions.
+<3> Require `io.pedestal.http` for the server _start_ and _stop_ functions.
+<4> Require `pedestal` for the Pedestal component.
+<5> Require `routes` for the application routes.
+
+Let's create a system initialization function named _app_.
+----
+include::component/src/system.clj[tags=app]
+----
+<1> It will have two arities to support the option of passing in
+start/stop functions.
+<2> The system map will contain a `:service-map` key who's value is
+a Pedestal service map.
+<3> We'll configure the service map with our app-specific routes.
+<4> The system map will contain a `:pedestal` key who's value is an
+uninitialized Pedestal component.
+<5> Pass the start/stop parameters to the Pedestal component
+constructor.
+<6> The Pedestal component depends on a service map, so will capture
+that dependency with `component/using`.
+
+== Running It
+We'll use link:http://boot-clj.com/:[boot-clj] to run our
+example. This should be familiar to you if you read through the
+link:hello-world[Hello World guide].
+
+Let's create a `build.boot` file so that we can fire up a boot repl.
+
+[[app-listing]]
+[source,clojure]
+.build.boot
+----
+include::component/build.boot[]
+----
+
+From the project's root directory, fire up a repl, and start the system.
+----
+$ boot repl
+...
+boot.user=> (require 'reloaded.repl)
+nil
+boot.user=> (require 'system)
+...
+boot.user=> (reloaded.repl/go)
+[nREPL-worker-2] INFO org.eclipse.jetty.util.log - Logging initialized @281159ms
+[nREPL-worker-2] INFO org.eclipse.jetty.server.Server - jetty-9.3.8.v20160314
+[nREPL-worker-2] INFO org.eclipse.jetty.server.handler.ContextHandler - Started o.e.j.s.ServletContextHandler@3017154a{/,null,AVAILABLE}
+[nREPL-worker-2] INFO org.eclipse.jetty.server.ServerConnector - Started ServerConnector@2ce2b722{HTTP/1.1,[http/1.1, h2c, h2c-17, h2c-16, h2c-15, h2c-14]}{0.0.0.0:8890}
+[nREPL-worker-2] INFO org.eclipse.jetty.server.Server - Started @281317ms
+:started
+----
+
+You can now interact with the started service.
+----
+ $ curl -i http://localhost:8890/greet
+
+HTTP/1.1 200 OK
+Date: Sun, 11 Sep 2016 23:16:24 GMT
+Strict-Transport-Security: max-age=31536000; includeSubdomains
+X-Frame-Options: DENY
+X-Content-Type-Options: nosniff
+X-XSS-Protection: 1; mode=block
+Content-Type: text/plain
+Transfer-Encoding: chunked
+Server: Jetty(9.3.8.v20160314)
+
+Hello, world!%
+----
+
+To stop the system, invoke the `reloaded.repl/stop` function from the
```

Comment:
Since the whole point of Component is to be able to start/stop/restart many times, it would be good to show that here.